src/azure-cli-testsdk/azure/cli/testsdk/preparers.py

```diff
@@ -109,7 +109,7 @@ def __init__(self, name_prefix='clitest.rg',  # pylint: disable=too-many-argumen
                  parameter_name_for_location='resource_group_location', location='westus',
                  dev_setting_name='AZURE_CLI_TEST_DEV_RESOURCE_GROUP_NAME',
                  dev_setting_location='AZURE_CLI_TEST_DEV_RESOURCE_GROUP_LOCATION'):
-        super(ResourceGroupPreparer, self).__init__(name_prefix, 90)
+        super(ResourceGroupPreparer, self).__init__(name_prefix, 75)
```

Comment:
Yes. Different services accept different lengths for resource group names, and in this instance 90 was rejected.